src/ngraph/attribute_adapter.cpp

```diff
@@ -31,388 +31,32 @@ using namespace ngraph;
 
 namespace ngraph
 {
-    // float
     constexpr DiscreteTypeInfo AttributeAdapter<float>::type_info;
-    const double& AttributeAdapter<float>::get()
```

Comment:
Why did you remove all `AttibuteAdapter`s?

Reply:
They are still there; same API, simpler template definition in the header.